tests/add_i128/Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2021"
 publish = false
 rust-version = "1.74"
 
+exclude = ["test_snapshots/"]
```

Comment:
The examples aren't published so it isn't necessary to add it here.

Reply:
Ah I was going off of `cargo package --list`, but I see the `publish = false` in the toml files. Fixed.